### ISSUE TITLE
On Pretrained AlexNet in MatConvNet

### ISSUE THREAD

**Issue by makedede** [vlfeat/matconvnet#680]
Hi, 

As the authors of MatConvNet mention, the AlexNet was trained using MatConvNet (beta17) and batch normalization with the code in the examples/imagenet. 

However, I find only pretrained parameters for conv layers but no parameters for batch norm layers. Can someone kindly explain?

If I would like fine tune the pretrained AlexNet, how should I do? Should I disable all batch-norm layers?

Thanks,

Kede
